src/widgets/search-box/search-box.js

```diff
@@ -25,8 +25,11 @@ const renderer = ({
   autofocus,
   searchOnEnterKeyPressOnly,
   wrapInput,
+  reset,
+  magnifier,
```

Comment:
I used the same naming as `poweredBy` option which accepts a boolean / object of template + css classes.

Using `showXXX` indicate me cannot be more than a boolean.